db/common_test.go

```diff
@@ -0,0 +1,143 @@
+// Package db - common test
+// The following file contains all the common functions for testing.
+package db
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"testing"
+
+	"github.com/packethost/pkg/log"
+)
+
+var (
+	ids = testID{
+		instanceID: "da013837-a30c-237a-adfc-2d9ad69d5000",
+		hardwareID: "ce2e62ed-826f-4485-a39f-a82bb74338e2",
+		templateID: "da013837-a30c-237a-adfc-2d9ad69d5011",
+		workflowID: "b356eede-22c1-11eb-8c17-0242ac120004",
+	}
+)
+
+func newCTX() ctx {
+	c, closer := context.WithCancel(context.Background())
+	return ctx{
+		context: c,
+		cancel:  closer,
+	}
+}
+
+func (a *testArgs) setCTX() {
+	a.ctx = newCTX()
+}
+
+func (a *testArgs) setLogger(t *testing.T) {
+	var err error
+	if a.logger, err = log.Init("github.com/tinkerbell/tink"); err != nil {
+		t.Error(err)
+	}
+}
+
+func (a *testArgs) setPsqlInfo() {
+	a.psqlInfo = fmt.Sprintf("host=%s port=%d user=%s password=%s dbname=%s sslmode=disable",
+		a.host, a.port, a.user, a.password, a.dbname)
+	a.logger.Info(a.psqlInfo)
+}
```

Comment:
similarly don't see too much of the point of these funcs but not a big deal to keep them in I think.